docs/Training-Domain-Variations.md

```diff
@@ -129,7 +126,9 @@ Once the class for the required method is specified, it must be registered in th
 This can be done by subscribing to the *register_sampler* method of the SamplerFactory. The command
 is as follows:
 
-`SamplerFactory.register_sampler(*custom_sampler_string_key*, *custom_sampler_object*)`
+```python
+SamplerFactory.register_sampler(*custom_sampler_string_key*, *custom_sampler_object*)
```

Comment:
In the rest of the documentation, we refer to "custom" arguments as `<argument_1>`. Maybe we should do the same here for consistency.

Reply:
Maybe. I didn't write this part. I will need to look over the contents a little more before making a change. The change here is just to formatting.

Reply:
I see.